www/front_src/src/Resources/Graph/Performance/Graph/index.tsx

```diff
@@ -553,7 +556,10 @@ const GraphContent = ({
             />
             {useMemoComponent({
               Component:
-                displayTimeValues && containsMetrics && position ? (
+                displayTimeValues &&
+                containsMetrics &&
+                position &&
+                !isOpenedModalAD ? (
```

Comment:
Why do you want to hide it when the anomaly detection modal is opened?

Reply:
https://github.com/centreon/centreon/pull/11544#pullrequestreview-1069260676 🤔

Reply:
Thank you for those explanations :)
What do you think about adding a prop on component that disables/enables those things. I think it's more reusable than integrating the atom which more related to the context ?

Reply:
ookk